lib/ansible/modules/cloud/amazon/aws_direct_connect_gateway.py

```diff
@@ -15,48 +15,45 @@
 module: aws_direct_connect_gateway
 author: Gobin Sougrakpam (@gobins)
 version_added: "2.5"
-short_description: Manage AWS Direct Connect Gateway.
+short_description: Manage AWS Direct Connect gateway.
```

Comment:
By convention, the `short_description` field does not end with a `.` - it's often not a complete sentence.
```suggestion
```